docs/topics/mta-7-configuration-web-console-on-ocp-local.adoc

```diff
@@ -0,0 +1,54 @@
+:_content-type: CONCEPT
+[id="mta-7-configuration-web-console-on-ocp-local_{context}"]
+= Configuration of the {ProductName} Operator in a {rh-ocp-local} environment
+
+The following table shows the different configurations of {rh-ocp-local} tested and the results obtained:
+
+[width="100%",cols="25%,25%,25%,25%",options="header",]
```

Comment:
I suggest just removing this table, this is too much information for the user. Additionally, we can't guarantee analysis times as there are too many factors to play. I would just give them a suggestion for what values to use; i.e. the last options in the table

Reply:
i.e. the 20/5/110. If we are saying that's the minimum reqs to get it working, then lets just push the user towards those values.

Reply:
@abrugaro && @dymurray - how about a small tweak to say recommended and keep the table in an abridged format

![image](https://github.com/user-attachments/assets/445e647b-2327-47c4-b87b-c4c902037908)

WDYT?

Reply:
If we want to se the minimum requisites I'd leave the following 2 rows, the other ones may add noise as @dymurray pointed

|Memory| CPU|Disk Size|
|---|---|---|
|20| 5| 110 |
|20| 5| 35, with the {ProductShortName} Operator configurations cache_data_volume_size and hub_bucket_volume_size set to 5GiB. |

Reply:
![image](https://github.com/user-attachments/assets/f4162657-5d91-4aba-9ae4-f20b9d62c6ee)